math/sigmoid.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <cmath>  // Untuk fungsi exp()
+using namespace std;
```

Comment:
```suggestion
#include <cassert>
```

atau bisa ganti dengan menggunakan header cassert untuk unittesting